src/styles/global/components.scss

```diff
@@ -0,0 +1,18 @@
+// Cart
+@import "src/components/cart/CartToggle/cart-toggle.scss";
+@import "src/components/cart/OrderSummary/order-summary.scss";
+
+// Sections
+@import "src/components/sections/BannerText/banner-text.scss";
+@import "src/components/sections/Breadcrumb/breadcrumb.scss";
+@import "src/components/sections/Hero/hero.scss";
+@import "src/components/sections/ProductDetails/product-details.scss";
+@import "src/components/sections/ScrollToTopButton/scroll-to-top-button.scss";
+@import "src/components/sections/Section/section.scss";
+
+// UI
+@import "src/components/ui/Badge/badge.scss";
+@import "src/components/ui/InputText/input-text.scss";
+@import "src/components/ui/SlideOver/overlay.scss";
+@import "src/components/ui/SROnly/sr-only.scss";
+@import "src/components/ui/SkuSelector/sku-selector.scss";
```

Comment:
I think this file should not exist in `gatsby.store`, I believe it came embedded in the patch

Reply:
hehehe my bad! I thought I had remove it.
Removed [here](https://github.com/vtex-sites/gatsby.store/pull/178/commits/c1945594dd91281a9df5a5253e60ab0b5dc5dc2c)